pkg/reconciler/wrappedPod.go

```diff
@@ -67,6 +67,10 @@ func getFlatIPSet(pod v1.Pod) map[string]void {
 
 	for _, network := range networkStatusList {
 		// we're only after multus secondary interfaces
+		if len(network.Interface) == 0 {
+			continue
+		}
+		// TODO: get whereabouts interface names from pod annotations
```

Comment:
I don't understand this comment.

Reply:
The multus secondary interfaces can be named by user. Maybe the interface names should be get from networks annotations. Otherwise, for custome interface name 'hello', ip reconciler will cleanup the ip and assign it to other pod.

BTW, a judgment for whether the interface is served by whereabout could be better.

Reply:
I still do not understand how we can have an interface within the pod named other than `netX` . Those are the names that will be reflected on the network status annotations.

Reply:
https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#launch-pod-with-text-annotation-with-interface-name

Reply:
I see. I was not aware of this, thanks for taking the time to lookup this document.

I'd rather you post an issue than adding a todo in the code.

Reply:
Thanks for your advice, i'll post an issue and remove the todo in the code.